packages/tasks/src/tasks/depth-estimation/inference.ts

```diff
@@ -17,19 +15,21 @@ export interface DepthEstimationInput {
 	/**
 	 * Additional inference parameters
 	 */
-	parameters?: DepthEstimationParameters;
+	parameters?: { [key: string]: unknown };
 	[property: string]: unknown;
 }
 
 /**
- * Additional inference parameters
- *
- * Additional inference parameters for Depth Estimation
+ * Outputs of inference for the Depth Estimation task
  */
-export interface DepthEstimationParameters {
+export interface DepthEstimationOutput {
+	/**
+	 * The predicted depth as an image
+	 */
+	depth?: unknown;
 	/**
-	 * When specified, limits the output to the top K most probable classes.
+	 * The predicted depth as a tensor
 	 */
-	topK?: number;
```

Comment:
Shouldn't we keep `topk` in parameters?

Reply:
It has been removed from the docstring in https://github.com/huggingface/transformers/pull/28733/files so I'm not sure

Ping @ydshieh who implemented the change in transformers

Reply:
Ah yes! You're right hehe